pilot/pkg/bootstrap/server.go

```diff
@@ -26,14 +26,15 @@ import (
 	"sync"
 	"time"
 
-	"istio.io/istio/pilot/pkg/networking/apigen"
-	"istio.io/istio/pilot/pkg/networking/grpcgen"
 	"k8s.io/client-go/kubernetes"
 	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/metadata"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
 
+	"istio.io/istio/pilot/pkg/networking/apigen"
+	"istio.io/istio/pilot/pkg/networking/grpcgen"
```

Comment:
move down

Reply:
I'll do make format in the smaller PR ( I'm moving stuff from this WIP to smaller PRs, as the design docs get approved).